pkg/ddl/index_cop_test.go

```diff
@@ -40,8 +40,11 @@ func TestAddIndexFetchRowsFromCoprocessor(t *testing.T) {
 		require.NoError(t, err)
 		tblInfo := tbl.Meta()
 		idxInfo := tblInfo.FindIndexByName(idx)
-		copCtx, err := copr.NewCopContextSingleIndex(tblInfo, idxInfo, tk.Session(), "")
+
+		sctx := tk.Session()
+		copCtx, err := ddl.NewReorgCopContext(store, ddl.NewDDLReorgMeta(sctx), tblInfo, []*model.IndexInfo{idxInfo}, "")
```

Comment:
Construct copCtx by `ddl.NewReorgCopContext` instead of `copr.NewCopContextSingleIndex` to make its inner state more like the one in production env.